distributed/protocol/tests/test_serialize.py

```diff
@@ -374,3 +374,16 @@ def dumps(f):
 
     msg = {"data": original}
     frames = await to_frames(msg)
+
+
+@pytest.mark.asyncio
+async def test_serialize_no_splitting():
+    cp = pytest.importorskip("cupy")
```

Comment:
I am not super happy that this requires cupy but I was unable to trigger a result with numpy.  Open to suggestions if folks have any

Reply:
Yeah there isn't one really as this is done correctly when using the `"dask"` serializer. We have to use something that uses the `"cuda"` serializer.